forc-pkg/src/manifest/mod.rs

```diff
@@ -550,7 +550,7 @@ impl PackageManifest {
     /// 2. The validity of the details provided. Makes sure that there are no mismatching detail
     ///    declarations (to prevent mixing details specific to certain types).
     pub fn validate(&self) -> Result<()> {
-        validate_name(&self.project.name, "package name")?;
+        validate_project_and_package_name(&self.project.name, "package name")?;
```

Comment:
> I don't think there is a difference between package name and project name. I agree that organization name should not have this validation.

What I mean is that this is still checking `self.project.name`. There's no need to pass in `use_case` as a string. We can just use "project name" as the string. The function can be renamed to `validate_project_name`

Reply:
@sdankel Done~